osbs/core.py

```diff
@@ -563,31 +563,32 @@ def watch_resource(self, resource_type, resource_name=None, **request_args):
             path += "%s/" % resource_name
         url = self._build_url(path, _prepend_namespace=False, **request_args)
 
-        while True:
-            with self._get(url, stream=True, headers={'Connection': 'close'}) as response:
-                check_response(response)
-                encoding = None
-                for line in response.iter_lines():
-                    logger.debug(line)
+        for retry in range(1, 10):
```

Comment:
Probably should be `range(10)`: len(range(1,10)) == 9 but len(range(10)) == 10.

Reply:
Fixed.